src/commands/spawn.ts

```diff
@@ -15,6 +18,7 @@ export default class Spawn extends Command {
 
   static flags = {
     env: Flags.string({ char: 'e', description: 'optional JSON object of environment variables to pass to the container' }),
+    port: Flags.string({ char: 'p', description: 'port for jamsocket to send requests to (default is 8080)' }),
```

Comment:
`Flags.integer` seems more appropriate

https://github.com/oclif/parser/blob/main/test/parse.test.ts#L329

This would save some of the type-checking below as well.

Reply:
Ah yeah, good call